### ISSUE TITLE
Update README to use milliseconds epoch in examples

### ISSUE THREAD

**Issue by fantapop** [spencermountain/spacetime#123]
Examples on the README are somewhat confusing as they're still using seconds epoch:

```
//epoch
s = spacetime(1489520157)
//array [yyyy, m, d] (zero-based months, 1-based days)
s = spacetime([2017, 5, 2])
//iso
s = spacetime('July 2, 2017 5:01:00')

// All inputs accept a timezone, as 2nd param:
s = spacetime(1489520157, 'Canada/Pacific')
s = spacetime('2019/05/15', 'Canada/Pacific')
``

**Comment by spencermountain** [comment]
ah!
thanks!

**Comment by spencermountain** [comment]
fixed in `5.7.0`
thanks
